go-controller/pkg/cluster/master.go

```diff
@@ -280,27 +269,27 @@ func (cluster *OvnClusterController) SetupMaster(masterNodeName string) error {
 	}
 
 	// Create 2 load-balancers for east-west traffic.  One handles UDP and another handles TCP.
-	k8sClusterLbTCP, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "load_balancer", "external_ids:k8s-cluster-lb-tcp=yes")
+	cluster.TCPLoadBalancerUUID, stderr, err = util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "load_balancer", "external_ids:k8s-cluster-lb-tcp=yes")
 	if err != nil {
 		logrus.Errorf("Failed to get tcp load-balancer, stderr: %q, error: %v", stderr, err)
 		return err
 	}
 
-	if k8sClusterLbTCP == "" {
-		stdout, stderr, err = util.RunOVNNbctl("--", "create", "load_balancer", "external_ids:k8s-cluster-lb-tcp=yes", "protocol=tcp")
+	if cluster.TCPLoadBalancerUUID == "" {
```

Comment:
Does it make sense to have a higher-level `EnsureLoadBalancer` function? I can't find a lot of documentation on all the properties the `load_balancer` can have.

Reply:
Maybe, but it's only 4 ovn-nbctl calls at the moment. Not sure it's worth it yet. We'll be re-doing some of this when we do DB reconciliation.